extensions/amp-fx-parallax/0.1/amp-fx-parallax.js

```diff
@@ -0,0 +1,223 @@
+/**
```

Comment:
Let's add a `article-parallax.amp.html` to `examples` folder.
Maybe: https://gist.github.com/aghassemi/0fde9b1df0ad531dbdfc29588ca27c73 (paths need updating)